lib/cartopy/io/shapereader.py

```diff
@@ -175,7 +181,20 @@ def geometry(self):
         return self._geometry
 
 
-class Reader(object):
+class FionaRecord(Record):
+    """
+    A single logical entry from a shapefile, combining the attributes with
+    their associated geometry. This is extends the standard Record to work
```

Comment:
This is -> This

Reply:
I have fixed the spelling too, and tried to rebase it into my master branch now.